pkgs/applications/audio/ChowCentaur/default.nix

```diff
@@ -0,0 +1,41 @@
+{ stdenv, lib, fetchFromGitHub, cmake, pkg-config, cairo, libxkbcommon, xcbutilcursor, xcbutilkeysyms, xcbutil, libXrandr, libXinerama, libXcursor, alsa-lib, libjack2 ,lv2, gcc-unwrapped
+}:
```

Comment:
```suggestion
{ stdenv
, lib
, fetchFromGitHub
, cmake
, pkg-config
, cairo
, libxkbcommon
, xcbutilcursor
, xcbutilkeysyms
, xcbutil
, libXrandr
, libXinerama
, libXcursor
, alsa-lib
, libjack2
, lv2
, gcc-unwrapped
}:
```

Reply:
I ran nixfmt on my PR and it changed the buildInputs to this format (one line per input), but it didn't change the above lines.
Should I change it anyway?
If yes, can nixfmt be improved so it does this?